J.Coate_H.Nguyen_Case_Study_1.Rmd

```diff
@@ -47,6 +53,10 @@ ggtitle("Breweries by State") + ylab("Count") + xlab("State")
 #### Created reproducible import code for Huy and Jaclyn to be able to reproduce on their local computers.
 #### Added nrow and ggplot to properly graph and answer question 1
 
+# Huy Completed:
+
+#### Number of breweries are present in each state? (Wev give exact number and after that we show by using bat plot for visualization. What do you think?)
+
```

Comment:
So do you want to do a bar graph AND a table? I'm okay with both. Is that what you added above?

Reply:
As you want. I think both because they asked us How many too.